CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # 3.1.4
 
-Fixes incorrect dropdown placement in IE.
+Fixes incorrect dropdown placement in IE. #1733
```

Comment:
I think we will need a new entry in the changelog for v3.1.5 and to update the package.json version to 3.1.5 and finally do a npm install. Otherwise we will have to do it in a separate PR